appinventor/components/src/ua/anatolii/graphics/ninepatch/BitmapType.java

```diff
@@ -0,0 +1,124 @@
+package ua.anatolii.graphics.ninepatch;
```

Comment:
Rather than pulling in the sources, I think it would be better if we just included a compiled version under lib/ instead. I don't expect we will be maintaining this code but just using it as a dependency.